meson.build

```diff
@@ -14,10 +14,16 @@ enable_rust = get_option('enable_rust')
 
 bpf_clang = find_program(get_option('bpf_clang'))
 
+enable_stress = get_option('enable_stress')
+
 if enable_rust
   cargo = find_program(get_option('cargo'))
 endif
 
+if enable_stress
+  run_stress_tests = find_program(join_paths(meson.current_source_dir(),
+                                      'meson-scripts/run_stress_tests'))
```

Comment:
nit: here and below, would you mind please fixing the alignment `meson-scripts/run_stress_tests` should align with `meson.current_source_dir()`)?

Reply:
Actually, given that you're going on PTO, we can just land this and fix in tree.